packages/blockchain-info-components/src/Form/SelectInput/template.js

```diff
@@ -49,11 +49,12 @@ const SelectInput = props => {
   } = props
   const options = items.map(item => ({ value: item.value, label: item.text }))
   const defaultValue = head(filter(x => equals(x.value, defaultItem), options))
-
+  // Give StyledSelect component isClearable prop
```

Comment:
This is also unnecessary to comment in the code itself, but maybe you want to comment on the PR?

Reply:
Thank you for the feedback. I will amend these commits and continue to work on #843